rixs/process2d.py

```diff
@@ -0,0 +1,550 @@
+""" Functions for processing 2D Image data.
+
+Typical command line workflow is executed in run_rest()
+
+Parameters
+----------
+selected_image_name : string
+    unique descriptor of the data that will be convereted
+    into a rixs spectrum
+photon_events : array
+    three column x, y, Iph photon locations and intensities
+curvature : dictionary
+    n2d order polynominal coeficients defining image curvature
+    {'x^2': coef, 'x1': coef, 'x0': coef}
+image_meta : string
+    container for metadata
+spectrum: array
+    binned spectrum two column defining
+    pixel, intensity
+resolution : array
+    parameters defining gaussian from fit_resolution
+"""
+
+import numpy as np
+from numpy.random import rand
+import matplotlib.pyplot as plt
+import lmfit
+
+
+def make_fake_image(x0=1000, x1=0.02, x2=0., sigma=2, noise=0.002):
```

Comment:
@mpmdean This is the function that I think should move to the test module

Reply:
Will do